docs/source/compile/programming_model.common_graph_breaks.md

```diff
@@ -0,0 +1,142 @@
+---
```

Comment:
In the image dynamo_coverage.png, suggesting to move the text under the image to the .md file - if something needs to be changed - it will be hard to do as noone has access to the source file.